fintech-examples/fintech-server/src/main/resources/application.yml

```diff
@@ -55,3 +55,8 @@ fintech-ui:
   redirect-url: http://localhost:4444/redirect-after-consent?redirectCode={redirectCode}
   unauthorized-url: http://localhost:4444/
   exception-url: http://localhost:4444/excaption-redirect
+
+security:
+  encoded-private-key: MIIEvwIBADANBgkqhkiG9w0BAQEFAASCBKkwggSlAgEAAoIBAQDRCFlWDkoqH1jNcVPyS5N2zFgeAna6z6J03Kb4vg/T07kGMlohk9vHuMEIp/wI0s0xX4e41rpo/ARMjIaxgb2t4kC6kD4XQRPHMNn/vTOmue17R4R+K3ADZ/LeWrRU7C1vY25pFAEBwlkL4OeLdQCNHht9ZuXD47NZKuGPzA2JzJGAT+i/xsWLKGphmsHHU1byhBxILkReZ6+7eedreLOUFtn/rc614cdlSZvK59buFToKT2Wr9w/hAwqaIwOPTbSm3CjH2DiEJzDLSIv6ohykZ7qB+4Ewy231dCdG7EhB45Dytira5pTqI6CPe9tJ6SrCs4K0p/8yigRwQZK8hr4TAgMBAAECggEBALMIJpPZK1ZGgOV80fxtFIEkZbhWPeCko6zZq7zR9HZmGUugZqfpnszBhbtnPStFeW93NaRvbTsr1nbFyTtRNucuKGhh0HxURZ9OTPiH0SxUhH0Y9PanN4oDRvWA1IEDw6pqjA89jds23NOarlY4QN+c0MioD9SmOl79pjFmAJZq7ULfLygHhfk4hj7UJGjfxqyg1mR3F3M7LhqDnPSi7MDNRSdkdgk87fR+SSlL39DSYmzTp1LLiyM92RgGSwJHMdl+PWicIFVRI5o7WMJ47+VFvHk+UFHzitsB+TIy7/SQ7x8ZBWTydrlb//g649dbSGXKnUuP8cymq//nXz5p5tECgYEA+Pw4wtvUQ4mrMwriNL367W0qEK/qnGaJTOKVFf6cqWs9hCizVKOPL6/V+5VeJ/1RWDGQR4jpD//Y01AsEylJvg8oYBhKtjHoSfET3zls/rLlS/d2h6bgxzWGJ8DDKPdsUKtR4s8YXl2da1Hm3U1ZhB5RX7clIBt+SEYr4yk7qBkCgYEA1uv5LwZPPLhM42MiLnzBKUWKMXFyCO2zK+C3gTkfcP4Ezdpml54XgdCyUKYJtTstd5ynU/9ZXvaXZ1GhJX4EkI+Jwck34Y1u30jMwzQfLGi/Z9+UHSlgBNJu92Q1sOQN8c+UdYRcWJmxQpITRSbLD35fuUC414Hf+w39Hcl/TQsCgYEAscr5hTuQkUGnPGWTLfJTUmNXXF8h+tjFRl+t8ezEVb48STj/BXtd7cFz3ZjFiWy8P2v1CP/c3Q3m+M4RCwW6oLsWK7hs8TkmFFbPx+Cr0hxQ3DQzMZ6ad+XijFR7h69WayxK5tjXH9tdY5sybeij+VHk7aJMImc/f8CL6kQxU1ECgYEAxYtUGMMF5dvI4GBoNK56pULDyLprhIVnYHdGdEJbAfKZ29RY5Bdtp+Z5X+OdnLTuD7kvm+di8ARcGZ1QLzPrYixtBG+9K1ZiiF1TyECKPrlw5Tr92ugC16vMTxwOta6ImrCf1nx4nP93jKbTi4YisvUHazFwNLWteAbdlpl0pw0CgYBxfCskpj8p69hM+VYtcY5eUFlnb5A3xPbH5COXO/O+GmwHRATJmVVZpULuO91IaMK0YWUIm6gkzHs+zp12DUpp0/T09AelQ43tmrXGx1fRNwzvjOliDTMHcii7f0zRZD+6uieZXsoPGmoFyWCxxmzrPP6ZjZiPoJX665JGrFqI5Q==
```

Comment:
Please add a comment that it is example, i.e.
```
 encoded-private-key: 
# This is the example private key value used only for DEMO purposes
MIIEvwIBADANBgkqhkiG9w0BAQEFAASCBKkwggSlAgEAAoIBAQDRCFlWDkoqH1jNcVPyS5N2zFgeAna6z6J03Kb4vg/T07kGMlohk9vHuMEIp/wI0s0xX4e41rpo/ARMjIaxgb2t4kC6kD4XQRPHMNn/vTOmue17R4R+K3ADZ
```

Reply:
done